src/App.js

```diff
@@ -23,56 +20,47 @@ class App extends Component {
 	}
 
 	componentDidMount = () => {
-		this.setState({ movies: movieData.movies });
-		// fetch from URL _____
-		//.then that parses the response object
-		// a .then that invokes a function in utils.js that cleans up the data to what we want to display
-		// Should set state
-		// Data will reflect:
-		// id
-		// movie
-		// poster_path
-		// backdrop_path
-		// release_date
-		// overview
-		// average_rating
-		// genres
-		// id
-		// name
-		// budget
-		// revenue
-		// runtime
-		// tagline
-	};
-
-	// showSelectedMovieDetails = (id) => {
-	// 	console.log(id);
-	// 	// Should hide banner and cards container
-	// 	// Should show movie details display
-	// };
-
-	hideSelectedMovieDetails = () => {
-		this.setState({ selectedMovie: {} });
+		fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Consider pulling your API calls into a separate file just like you did in mods 1 + 2.